clang/lib/Format/ContinuationIndenter.cpp

```diff
@@ -925,7 +925,10 @@ void ContinuationIndenter::addTokenOnCurrentLine(LineState &State, bool DryRun,
                         TT_TableGenDAGArgOpenerToBreak) &&
       !(Current.MacroParent && Previous.MacroParent) &&
       (Current.isNot(TT_LineComment) ||
-       Previous.isOneOf(BK_BracedInit, TT_VerilogMultiLineListLParen)) &&
+       (Previous.is(BK_BracedInit) &&
+        (!Style.Cpp11BracedListStyle || !Previous.Previous ||
+         Previous.Previous->isNotOneOf(tok::identifier, tok::l_paren))) ||
```

Comment:
```suggestion
         Previous.Previous->isNoneOf(tok::identifier, tok::l_paren))) ||
```

Reply:
This I have already done, but apparently forgot to add before the push.